compiler/model-import-verifier/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+set(SRCS_EVAL_TESTER
+      src/EvalDriver.cpp
+      src/sample_model.h
+      src/sample_model.cpp
+   )
+
+add_executable(model_eval_verifier ${SRCS_EVAL_TESTER})
+target_link_libraries(model_eval_verifier PRIVATE luci_interpreter_import)
+target_link_libraries(model_eval_verifier PRIVATE luci_interpreter)
+target_link_libraries(model_eval_verifier PRIVATE safemain)
+
+install(TARGETS model_eval_verifier DESTINATION bin)
```

Comment:
(optional) We used `tflite` or `circle` or event `caffe` for specific type of model not general `model` that means, one may think this `model-import-verifier` can be any model import verifier, which I don't think you intend to do. Or do you?

Reply:
I don't plan to introduce `model-import-verifier` module. I provide this only as example and simple proof  for this draft.

Reply:
You mean "land"? You already "introduced"(?) the module through this draft.
I think landing this module will actually "verify" if you revise this with gtest.
As of current draft, I do not see any tests for new codes.

Reply:
I agree with you: better to rewrite with gtest, but let me please add this change later, when code of other changes (code from `luci` and `luci-interpreter`) will looks OK for reviewers.